src/assets/stack-blitz/package.json

```diff
@@ -1,40 +1,42 @@
 {
-  "name": "material-aio-example",
+  "name": "example-app",
   "version": "0.0.0",
   "scripts": {
     "ng": "ng",
     "start": "ng serve",
     "build": "ng build",
-    "test": "ng test",
-    "lint": "ng lint",
-    "e2e": "ng e2e"
+    "watch": "ng build --watch --configuration development",
+    "test": "ng test"
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^13.0.0-next.0",
-    "@angular/cdk": "^13.0.0-next.0",
-    "@angular/common": "^13.0.0-next.0",
-    "@angular/compiler": "^13.0.0-next.0",
-    "@angular/core": "^13.0.0-next.0",
-    "@angular/forms": "^13.0.0-next.0",
-    "@angular/material": "^13.0.0-next.0",
-    "@angular/material-moment-adapter": "^13.0.0-next.0",
-    "@angular/platform-browser": "^13.0.0-next.0",
-    "@angular/platform-browser-dynamic": "^13.0.0-next.0",
-    "@angular/router": "^13.0.0-next.0",
-    "moment": "^2.29.1",
-    "rxjs": "^6.6.7",
-    "tslib": "^2.2.0",
-    "zone.js": "^0.11.4"
+    "@angular/animations": "13.0.0-rc.0",
+    "@angular/cdk": "${version}",
+    "@angular/common": "13.0.0-rc.0",
+    "@angular/compiler": "13.0.0-rc.0",
+    "@angular/core": "13.0.0-rc.0",
+    "@angular/forms": "13.0.0-rc.0",
+    "@angular/material": "${version}",
+    "@angular/platform-browser": "13.0.0-rc.0",
+    "@angular/platform-browser-dynamic": "13.0.0-rc.0",
+    "@angular/router": "13.0.0-rc.0",
+    "rxjs": "~6.6.0",
+    "sass": "1.35.1",
+    "tslib": "^2.1.0",
+    "zone.js": "~0.11.4"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^13.0.0-next.0",
-    "@angular/cli": "^13.0.0-next.0",
-    "@angular/compiler-cli": "^13.0.0-next.0",
-    "@angular/language-service": "^13.0.0-next.0",
-    "@angular/localize": "^13.0.0-next.0",
-    "@types/node": "^14.14.22",
-    "ts-node": "^8.10.2",
-    "typescript": "~4.3.5"
+    "@angular-devkit/build-angular": "13.0.0-rc.0",
+    "@angular/cli": "13.0.0-rc.0",
+    "@angular/compiler-cli": "13.0.0-rc.0",
+    "@types/jasmine": "3.10.1",
+    "@types/node": "^12.11.1",
```

Comment:
Why is this moved back to v12?

Reply:
this is what the CLI generates and I want to match that.